examples/with-google-analytics/pages/_document.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Document, { Head, Main, NextScript } from 'next/document'
+
+import { GA_TRACKING_ID } from '../lib/gtag'
+
+export default class extends Document {
+  static getInitialProps ({ renderPage }) {
+    return renderPage()
```

Comment:
const { html, head, errorHtml, chunks } = renderPage()
return { html, head, errorHtml, chunks, styles }

lets do it like in the readme

Reply:
Makes sense. Done ✅.